### PR TITLE
Eon65

### PR DESCRIPTION
**Pull request by Maartenwut**
## Description

Adds QMK support for the Eon65 PCB. It also has a VIA keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/maartenwut/eon65/config.h [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4705
+#define PRODUCT_ID      0xaeb4
```
Comment (fauxpark) [inline keyboards/maartenwut/eon65/config.h:23]:
```suggestion
#define PRODUCT_ID      0xAEB4
```

File: keyboards/maartenwut/eon65/config.h [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+#define LCD_DATA3_PIN    7            //< pin for 4bit data bit 3
+#define LCD_RS_PORT      LCD_PORT     //< port for RS line
+#define LCD_RS_PIN       3            //< pin  for RS line
+#define LCD_RW_PORT      LCD_PORT     //< port for RW line
+#define LCD_RW_PIN       2            //< pin  for RW line
+#define LCD_E_PORT       LCD_PORT     //< port for Enable line
+#define LCD_E_PIN        1            //< pin  for Enable line
+#endif
+*/
+
```
Comment (fauxpark) [inline keyboards/maartenwut/eon65/config.h:245]:
```suggestion
```

File: keyboards/maartenwut/eon65/rules.mk [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/maartenwut/eon65/rules.mk:33]:
```suggestion
```

**Comment by Maartenwut** [comment]
Merge, please? :)

File: keyboards/maartenwut/eon65/info.json [hunk 4]
```diff
@@ -0,0 +1,388 @@
...
+                {"x":3.75, "y":4, "w":6.25},
+                {"x":10, "y":4},
+                {"x":11, "y":4},
+                {"x":12, "y":4},
+                {"x":13, "y":4},
+                {"x":14, "y":4},
+                {"x":15, "y":4}
+            ]
+        },
+        "LAYOUT_65_ansi": {
```
Comment (fauxpark) [inline keyboards/maartenwut/eon65/info.json:87]: All these layouts (except `LAYOUT_all`) are missing a key vs the LAYOUT macros.

File: keyboards/maartenwut/eon65/info.json [hunk 5]
```diff
@@ -0,0 +1,388 @@
...
+                {"x":3.25, "y":3},
+                {"x":4.25, "y":3},
+                {"x":5.25, "y":3},
+                {"x":6.25, "y":3},
+                {"x":7.25, "y":3},
+                {"x":8.25, "y":3},
+                {"x":9.25, "y":3},
+                {"x":10.25, "y":3},
+                {"x":11.25, "y":3},
+                {"x":12.25, "y":3, "w":2.75},
```
Comment (noroadsleft) [inline keyboards/maartenwut/eon65/info.json:147]:
```suggestion
                {"x":12.25, "y":3, "w":1.75},
                {"x":13.25, "y":3},
```

File: keyboards/maartenwut/eon65/info.json [hunk 6]
```diff
@@ -0,0 +1,388 @@
...
```
Comment (noroadsleft) [inline keyboards/maartenwut/eon65/info.json:222]:
```suggestion
                {"x":12.25, "y":3, "w":1.75},
                {"x":13.25, "y":3},
```

File: keyboards/maartenwut/eon65/info.json [hunk 7]
```diff
@@ -0,0 +1,388 @@
...
```
Comment (noroadsleft) [inline keyboards/maartenwut/eon65/info.json:297]:
```suggestion
                {"x":12.25, "y":3, "w":1.75},
                {"x":13.25, "y":3},
```

File: keyboards/maartenwut/eon65/info.json [hunk 8]
```diff
@@ -0,0 +1,388 @@
...
```
Comment (noroadsleft) [inline keyboards/maartenwut/eon65/info.json:373]:
```suggestion
                {"x":12.25, "y":3, "w":1.75},
                {"x":13.25, "y":3},
```

File: keyboards/maartenwut/eon65/info.json (hunk 4 above)
Comment (Maartenwut) [reply keyboards/maartenwut/eon65/info.json:87]: Thanks for catching that!

**Comment by Maartenwut** [comment]
Thanks!

**Pull request merged**